RecoVertex/V0Producer/src/V0Fitter.h

```diff
@@ -46,6 +46,8 @@ class dso_hidden V0Fitter {
               reco::VertexCompositeCandidateCollection& l);
 
 private:
+  const edm::ESGetToken<MagneticField, IdealMagneticFieldRecord> esToken_MF;
```

Comment:
almost all data members use the (recommended) trailing underscore in the name. Please consider to update here as well.

Reply:
done in the last push.